src/app.js

```diff
@@ -17,6 +17,21 @@ const logger = require('../logs/logger');
 const app = express();
 require('express-async-errors');
 
+// Serve the Swagger JSON file
+// app.get('/swagger.json', (req, res) => {
+//   res.sendFile('swagger-output.json');
+// });
+
+// Serve Swagger UI static files
+app.use(
+  '/swagger-ui',
```

Comment:
Maybe make this route `api-docs` ?

Reply:
Renamed

Reply:
commit 87eeca14419b5365397d727763dcfd0e4a8d72d9